Star Trek Adventures 2d20/README.md

```diff
@@ -0,0 +1,39 @@
+# Star Trek Adventures 2d20 Character Sheet
+This is the for use on Roll20.net with the Modiphius' Star Trek Adventures. The sheet is designed to be usable by all Roll20 users regardless of subscription level.
+
+## Features
+
+* High performance worker scripts used throughout the sheet
+* Styling that matches the official sheet from Modiphius 
+* Character or Starship sheets available
+* Challenge Rolls pre-configured including setting difficulty of the task, adding additional dice rolls, and incorporating a trait - all styled in the Star Trek LCARS format
+
+
+[![Sheet]( https://raw.githubusercontent.com/Roll20/roll20-character-sheets/master/Star Trek Adventures 2d20/Star Trek Adventures 2d20.PNG "Star Trek Adventures 2d20")](https://raw.githubusercontent.com/Roll20/roll20-character-sheets/master/Star Trek Adventures 2d20/Star Trek Adventures 2d20.PNG)
+
+### [Changelog](https://raw.githubusercontent.com/Roll20/roll20-character-sheets/master/Star Trek Adventures 2d20/changelog.md)
+
+## Sheet Releases
+When I release a new version, it takes time to reach the public. Roll20 must merge my changes (typically Mondays/Tuesdays) and then release the code to the public (typically Tuesdays/Wednesdays). You can track [Roll20's pull requests on their github](https://github.com/Roll20/roll20-character-sheets/pulls). I have no control over when they release.
+
+## How to Update the sheet yourself (as Pro):
+
+1. Copy the [html](https://raw.githubusercontent.com/GrumpyOldDM/STA_DemoSheet/master/Star%20Trek%20Adventures%202d20.html) from GitHub
+2. In Roll20 go to campaign settings and choose a custom character sheet.
+3. In the HTML tab paste the HTML code
+4. Copy the [css](https://raw.githubusercontent.com/GrumpyOldDM/STA_DemoSheet/master/Star%20Trek%20Adventures%202d20.css) from GitHub
+5. On Roll20 in the CSS tab paste the CSS code.
+6. Hit save.
+
+*If you are ever concerned about data loss I recommend you duplicate your campaign and use that to copy to see how it'll work. *
+
+## Testing new versions
+I regularly release new versions. If you'd like to be involved in the testing period follow this thread. You can also see the latest code on [Star Trek Adventures (Workshop)](https://app.roll20.net/join/2363663/M4ln3Q)
+
+## Contributors
+This sheet wouldn't be possible without the help of the following users:
+Rob - https://app.roll20.net/users/28243/rob 
```

Comment:
Would also appreciate a mention here. I wish there was an easier way to cherry pick commits from one repo to another, since it looks like all my commits are gone.